README.md

```diff
@@ -0,0 +1,72 @@
+# IOXIO Dataspace example app
+
+This demo application is designed to show a practical example on how to create an
+application using Dataspace tecnhologies.
+
+It consists of simple Python backend that is responsible for authentication and data
+retrieval and React-based frontend application.
+
+You can check the [online demo](TODO: add a link there) or run it locally (this way
+requires some configuration, so check [configuration](#configuration) section below).
+
+Main idea is to demostrate how to:
+
+- Perform authentication in a dataspace
+- Handle application configuration correctly
+- Retrieve data products from Product Gateway
+
+## Repo structure
+
+- [backend](./backend/) - FastAPI backend
+  - [main.py](./backend/app/main.py) - All the backend routes, e.g for authentication or
+    data retrieval
+  - [settings.py](./backend/app/settings.py) - Backend configuration
+- [frontend](./frontend) - React application
+  - [containers](./frontend/src/containers) - Root containers for handling data products
+  - [components](./frontend/src/components) - Stateless components to simplify following
+    the containers' logic
+  - [utils](./frontend/src/utils) - Some helpers, e.g for making network requests to
+    backend
+
+## Local installation
+
+### Configuration
+
+Before running the app locally, you have to:
+
+1. Register an application in
+   [Developer Portal](https://developer.sandbox.ioxio-dataspace.com/)
```

Comment:
It would be great if this would show details on how to fill in the `Redirect URIs` and `Logout Redirect URIs`, since the paths fro those are not necessarily obvious without reading the source code in detail.